blimflix/client/app/core/sidebar.scss

```diff
@@ -0,0 +1,75 @@
+.sidebar-wrapper {
+height: 100%;
+background-color: #222;
+position: fixed;
+left: 0;
+top: 50px;
+padding: 0;
+padding-top: 70px;
+border-right: 2px solid black;
+}
+
+.profile-wrapper {
+height: 50px;
+color: #FFFFFF;
+}
+
+.row {
+    color: #FFFFFF;
+}
+
+.row img {
+    width: 100%;
+    height: 100%;
+}
+
+.row h4 {
+    font-weight: bold;
+}
+
+.row h4, .row h5 {
+    padding: 0;
+    margin:  0;
+    padding-top: 5px;
+    padding-left: 10px
+}
+
+.col-centered{
```

Comment:
Hi team, let's not override bootstrap directly.

What we can do instead is to define a class with the overridden code and add it to the element involved, chances are that the other views may need to have the original behavior.